packages/react-component-library/src/components/Autocomplete/AutocompleteOption.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import reactStringReplace from 'react-string-replace'
 
-import { SelectBaseOption, SelectBaseOptionAsStringProps } from '../SelectBase'
+import { SelectBaseOption, SelectBaseOptionProps } from '../SelectBase'
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AutocompleteOptionProps extends SelectBaseOptionAsStringProps {
+export interface AutocompleteOptionProps extends SelectBaseOptionProps {
   inputValue?: string
 }
 
 export const AutocompleteOption = React.forwardRef<
   HTMLLIElement,
   AutocompleteOptionProps
 >(({ children, inputValue, ...rest }, ref) => {
+  const optionElement =
+    inputValue && typeof children === 'string'
```

Comment:
I had to add type string to appease ts 
I was getting `Type 'null' is not assignable to type 'string | ReactNodeArray | undefined'.`

Reply:
Should be able to omit it if `AutocompleteOptionProps` is made to extend from `SelectBaseOptionAsStringProps` again.

Reply:
👍